docs/framework/unmanaged-api/wmi/putinstancewmi.md

```diff
@@ -2,57 +2,59 @@
 title: PutInstanceWmi function (Unmanaged API Reference)
 description: The PutInstanceWmi function creates or updates an instance of an existing class.
 ms.date: "11/06/2017"
-api_name: 
+api_name:
   - "PutInstanceWmi"
-api_location: 
+api_location:
   - "WMINet_Utils.dll"
-api_type: 
+api_type:
   - "DLLExport"
-f1_keywords: 
+f1_keywords:
   - "PutInstanceWmi"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "PutInstanceWmi function [.NET WMI and performance counters]"
-topic_type: 
+topic_type:
   - "Reference"
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
+
 # PutInstanceWmi function
-Creates or updates an instance of an existing class. The instance is written to the WMI repository. 
+
+Creates or updates an instance of an existing class. The instance is written to the WMI repository.
 
 [!INCLUDE[internalonly-unmanaged](../../../../includes/internalonly-unmanaged.md)]
-  
-## Syntax  
-  
-```  
+
+## Syntax
+
+```cpp
 HRESULT PutInstanceWmi (
    [in] IWbemClassObject*    pInst,
    [in] long                 lFlags,
    [in] IWbemContext*        pCtx,
    [out] IWbemCallResult**   ppCallResult
-); 
-```  
+);
+```
 
 ## Parameters
 
-`pInst`    
-[in] A pointer to the instance to be writen.
+`pInst`
+[in] A pointer to the instance to be written.
 
-`lFlags`   
-[in] A combination of flags that affect the behavior of this function. The following values are defined in the *WbemCli.h* header file, or you can define them as constants in your code: 
+`lFlags`
+[in] A combination of flags that affect the behavior of this function. The following values are defined in the *WbemCli.h* header file, or you can define them as constants in your code:
 
 |Constant  |Value  |Description  |
 |---------|---------|---------|
-| `WBEM_FLAG_USE_AMENDED_QUALIFIERS` | 0x20000 | If set, WMI does not store any qualifiers with the **Amended** flavor. <br> If not set, it is assumed that this object is not localized, and all qualifiers are storedwith this instance. |
+| `WBEM_FLAG_USE_AMENDED_QUALIFIERS` | 0x20000 | If set, WMI does not store any qualifiers with the **Amended** flavor. <br> If not set, it is assumed that this object is not localized, and all qualifiers are stored with this instance. |
 | `WBEM_FLAG_CREATE_OR_UPDATE` | 0 | Create the instance if it does not exist, or overwrite it if it exists already. |
 | `WBEM_FLAG_UPDATE_ONLY` | 1 | Update the instance. The instance must exist for the call to be successful. |
 | `WBEM_FLAG_CREATE_ONLY` | 2 | Create the instance. The call fails if the instance already exists. |
 | `WBEM_FLAG_RETURN_IMMEDIATELY` | 0x10 | The flag causes a semisynchronous call. |
 
-`pCtx`  
-[in] Typically, this value is `null`. Otherwise, it is a pointer to an [IWbemContext](/windows/desktop/api/wbemcli/nn-wbemcli-iwbemcontext) instance that can be used by the provider that is providing the requested classes. 
+`pCtx`
```

Comment:
```suggestion
`pCtx`\
```

Reply:
I missed that you were using the double space return. I'm wondering if these should these be H3s in these docs?

Reply:
It's something worth investigating. I was going for a similar look and feel of the managed reference pages.

Reply:
Also, the double space return is a migration hack that we're trying to get rid of over time. So I like that we're cleaning the extra spaces.